From 3e9ba2f9cc7b5e954a4d757fc29572bbce397e99 Mon Sep 17 00:00:00 2001
From: Johannes Lorenz <1042576+JohannesLorenz@users.noreply.github.com>
Date: Tue, 13 Dec 2022 22:22:44 +0100
Subject: [PATCH 1872/1936] Fix sympathetic effect multiplier (#219)

Analogous implementation to
4e0b22f267f4254ad9deed74856d9155aef55848, but this time, for the
Sympathetic effect.
---
 src/Effects/Sympathetic.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Effects/Sympathetic.cpp b/src/Effects/Sympathetic.cpp
index 1380af9f..7a6bd16e 100644
--- a/src/Effects/Sympathetic.cpp
+++ b/src/Effects/Sympathetic.cpp
@@ -284,7 +284,7 @@ unsigned char Sympathetic::getpresetpar(unsigned char npreset, unsigned int npar
     if(npreset < NUM_PRESETS && npar < PRESET_SIZE) {
         if(npar == 0 && insertion == 0) {
             /* lower the volume if this is system effect */
-            return (3 * presets[npreset][npar]) / 2;
+            return (2 * presets[npreset][npar]) / 3;
         }
         return presets[npreset][npar];
     }
-- 
2.47.0

